janus_server/src/bin/aggregation_job_driver.rs

```diff
@@ -705,6 +715,76 @@ impl AggregationJobDriver {
             .await?;
         Ok(())
     }
+
+    // XXX: log that we are canceling?
```

Comment:
This appears to be stale, we are logging at the callsite.

Reply:
Gah, right you are, thanks.